lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -2064,7 +2064,8 @@ class ByteCodeBufferBuilder
                 | FunctionInfo::Attributes::ClassMethod
                 | FunctionInfo::Attributes::Method
                 | FunctionInfo::Attributes::EnclosedByGlobalFunc
-                | FunctionInfo::Attributes::AllowDirectSuper)) == 0,
+                | FunctionInfo::Attributes::AllowDirectSuper
+                | FunctionInfo::Attributes::ComputedName)) == 0,
             "Only the ErrorOnNew|SuperReference|Lambda|CapturesThis|Generator|ClassConstructor|BaseConstructorKind|Async|ClassMember|Method|EnclosedByGlobalFunc|AllowDirectSuper attributes should be set on a serialized function");
```

Comment:
Looks like this error message also should be updated to include `ComputedName`

Reply:
done.